src/views/shared/workflow-status-tag/workflow-status-tag.types.ts

```diff
@@ -1,12 +1,6 @@
-export type WorkflowStatus =
-  | 'WORKFLOW_EXECUTION_CLOSE_STATUS_INVALID'
-  | 'WORKFLOW_EXECUTION_CLOSE_STATUS_COMPLETED'
-  | 'WORKFLOW_EXECUTION_CLOSE_STATUS_FAILED'
-  | 'WORKFLOW_EXECUTION_CLOSE_STATUS_CANCELED'
-  | 'WORKFLOW_EXECUTION_CLOSE_STATUS_TERMINATED'
-  | 'WORKFLOW_EXECUTION_CLOSE_STATUS_CONTINUED_AS_NEW'
-  | 'WORKFLOW_EXECUTION_CLOSE_STATUS_TIMED_OUT'
-  | 'WORKFLOW_EXECUTION_STATUS_RUNNING';
+import { type WorkflowExecutionCloseStatus } from '@/__generated__/proto-ts/uber/cadence/api/v1/WorkflowExecutionCloseStatus';
+
+export type WorkflowStatus = WorkflowExecutionCloseStatus;
```

Comment:
Should this be removed?

Reply:
The intent is to keep the types in Web separate from backend types, in case we need to make any modifications in the future.